test/e2e/multihoming.go

```diff
@@ -1328,6 +1328,203 @@ var _ = Describe("Multi Homing", func() {
 					),
 				),
 			)
+
+			ginkgo.DescribeTable(
+				"multi-network ingress allow all",
+				func(netConfigParams networkAttachmentConfigParams, clientPodConfig podConfiguration, serverPodConfig podConfiguration, policy *mnpapi.MultiNetworkPolicy) {
+					netConfig := newNetworkAttachmentConfig(netConfigParams)
+
+					By("setting up the localnet underlay")
+					nodes := ovsPods(cs)
+					Expect(nodes).NotTo(BeEmpty())
+					defer func() {
+						By("tearing down the localnet underlay")
+						Expect(teardownUnderlay(nodes)).To(Succeed())
+					}()
```

Comment:
shoudn't this be done only if you managed to setup the underlay  ?

Reply:
yes and no, 
lets say you created some bridges and then failed, so it will remove the ones that it created

since it was already the state before, i just copy pasted tbh due to the above reason